README.md

```diff
@@ -166,7 +168,68 @@ The airdrop has two rounds: In the first round, all participants will go through
+{
+  "type": "team race round",
+  "flow": "the flow id, check flow with flow-id on https://github.com/game-of-nfts/gon-testnets/blob/main/doc/flow-table.md",
+  "last_owner": "the ultimate owner of this NFT",
+  "start_height": "transfer before this height are considered invalid"
+}
+```
+
+**Rules:**
+
+- The `start_height` is the block height on IRISnet. Any transfer prior to the `start_height` will be considered invalid, which means your first Interchain NFT Transfer must happen after that height. Note that `start_height` is not used for calculate your grade.
+- The `last_owner` is an official address on IRISnet, and is the final recipient of the NFT in this game. 
+- Considering the time zone difference, the completion time is calculated by the **difference** in height between the last NFT transfer (to the `last_owner`) and the first Interchain NFT transfer (not `start_height`) on IRISnet.
+- Participants must use their own account addresses provided at the registration stage.
+- Participants must complete all of B5, B6 and B7 in order to be eligible for the ranking, which is calculated by summing the differences in block heights of each NFT between the last NFT transfer and the first Interchain NFT transfer on IRISnet.
+- The third round of the competition will conclude at `2023-03-24 23:59:59 UTC`. After this, you can still complete B5, B6 and B7, but your race rank will not be counted, and you will not be eligible to win B8.
+- The top 10 participants in each airdrop round will be awarded B8 points. Evidence is not required for B8.
+
+#### **Round 5: Quiz Game**
+
+In the quiz game, the organizers will airdrop 100 Quiz NFTs that are owned by 100 preset addresses around `2023-03-21 08:00 UTC`. Participants are required to **guess the answer** to the question in `Task Data`, which serves as the secret key to decrypt the mnemonics. The participants must then use the mnemonics to generate these preset addresses and transfer the NFT according to the flow.
+
+The quiz questions cover different domains which are related to the Cosmos world. All answers are single words containing lowercase characters, hyphens, and Arabic numerals. 
+
+**Task Data:**
+
+```json
+{
+	"question": "the answer of this question is the key to decrpyt the mnemonics",
+	"mnemonics_encrypted": "aes encrypted mnemonics",
+	"flow": "the flow id, check the flow off-chain"
+}
+```
+
+**Rules:**
+- The answer is a single word of lowercase characters, hyphens and numbers. The possible format are `abc`, `abc-10`, `10-abc`, `10abc`, `abc10` and `1234`.
+- To ensure fairness, any DMs to the coordinators regarding quiz game will not be responded to.
+- A uniform encryption and decryption tool is provided to ensure that the answers can be correctly decrypted.
+- The total number of NFTs is only 100, and each player can solve as many questions as possible to get points.
+- The final owner of the NFT must be participants’ own addresses on IRISnet. Otherwise, they won’t get points.
```

Comment:
Does this mean we do the flow with the original address, then _after_ the flow, we transfer it to ourselves?

Reply:
The key is to take control of the original address. The ultimate owner must be your iris address. As for validation, since these NFTs follow a never-go-back style, their class trace is fixed on the IRIS side. We only query the IBC/Class and determine who owns the NFT under them.